elasticsearch/utils/logging

```diff
@@ -17,7 +17,7 @@ esac
 
 # the deployment mounts the secrets at this location - not necessarily the same
 # as $ES_PATH_CONF
-secret_dir=/etc/elasticsearch/secret
```

Comment:
if we change this then the configmap will be incorrect https://github.com/openshift/elasticsearch-operator/blob/feature-es6x/pkg/k8shandler/configuration_tmpl.go#L38